package/rdspgbadger.py

```diff
@@ -114,7 +114,8 @@ def get_all_logs(dbinstance_id, output,
 def write_log(client, dbinstance_id, filename, logfilename):
     response = client.download_db_log_file_portion(
         DBInstanceIdentifier=dbinstance_id,
-        LogFileName=logfilename
+        LogFileName=logfilename,
+        Marker="0"
```

Comment:
I have trouble to see how it can have bad side effect not setting it, but I can't see the harm of being explicit :+1:

Reply:
It turns out to be required to make downloads bigger than 1MB to work, see PR description.